server/src/main/java/com/thoughtworks/go/server/service/GoConfigService.java

```diff
@@ -1043,10 +1042,68 @@ public ArtifactStores artifactStores() {
         return cruiseConfig().getArtifactStores();
     }
 
+    public CruiseConfig validateCruiseConfig(CruiseConfig cruiseConfig) throws Exception {
```

Comment:
Except for this method, just moves in this file?

Reply:
Exactly correct :)